src/core/commandVersionUpgrades/canonicalizeAndValidateCommand.ts

```diff
@@ -142,10 +138,7 @@ export async function checkForOldInference(
       );
 
       if (pressed) {
-        graph.extensionContext.globalState.update(
-          globalStateKeys.hideInferenceWarning,
-          true
-        );
+        graph.ide.globalState.set("hideInferenceWarning", true);
```

Comment:
Note that the value is now strongly typed, so you have to put a boolean here for this key